mvt/feature_test.go

```diff
@@ -1,143 +1,128 @@
 package mvt
 
 import (
+	"fmt"
 	"testing"
 
 	"context"
 
+	"github.com/gdey/tbltest"
 	"github.com/terranodo/tegola"
 	"github.com/terranodo/tegola/basic"
 	"github.com/terranodo/tegola/mvt/vector_tile"
 )
 
 func TestEncodeGeometry(t *testing.T) {
-	/*
-		complexGemo := basic.Polygon{
-			basic.Line{
-				basic.Point{8, 8.5},
-				basic.Point{9, 9},
-				basic.Point{20, 20},
-				basic.Point{11, 20},
-			},
-		}
-	*/
-	testcases := []struct {
+	type tc struct {
 		desc string `tbltest:"desc"`
 		geo  basic.Geometry
 		typ  vectorTile.Tile_GeomType
 		bbox tegola.BoundingBox
 		egeo []uint32
 		eerr error
-	}{
-		{ //0
-			geo: nil,
-			typ: vectorTile.Tile_UNKNOWN,
-			bbox: tegola.BoundingBox{
-				Minx: 0,
-				Miny: 0,
-				Maxx: 4096,
-				Maxy: 4096,
-			},
+	}
+	tile := tegola.NewTile(20, 0, 0)
+	fromPixel := func(x, y float64) *basic.Point {
+		pt, err := tile.FromPixel(tegola.WebMercator, [2]float64{x, y})
+		if err != nil {
+			panic(fmt.Sprintf("error trying to convert %v,%v to WebMercator. %v", x, y, err))
```

Comment:
why not t.Errorf(...)?

Reply:
Because this is an error with the test-case and not the function being tested. I want to panic, as the test case code is bad. And any result whether good or bad is suspect.

Reply:
Look at the code further, looks like I'm using it as a helper to build out the test cases, so returning an error is a no for the use case. Leaving for now, till I can think of a better system for building out the testcases.